db/migrate/20240708153903_create_service_translations.rb

```diff
@@ -0,0 +1,20 @@
+require 'csv'
+
+class CreateServiceTranslations < ActiveRecord::Migration[7.1]
+  def change
+    create_table :service_translations do |t|
+      t.string :key
+      t.string :translation
+      t.timestamps
```

Comment:
if we are going to do this I would have preferred a translations table and then have a translation_type field of service_type

would make it more generic if we need it in the future.

Reply:
I think this is a good shout - I could say that we don't really want to keep it like this long term and it would be good to tack it into it's own gem like we discussed about things like this and fee calculations - BUT given we may not get that time I think it's worth doing now

Reply:
@ivanELEC and @dwhenry I'm really reluctant to do this temporarily and I would like to discuss this during the team day. 

I think it would be useful to have an ID for each of these translations as well as the key? We could then use that ID in the json blob and for each client application. 

And in theory the value for the translation should be in a separate table linked via that ID and translation_type should be an id linked to a separate normalised translation type table. 

But can we keep this on hold and look in detail on team day. For now we just use the key